games/applications/run_bv.py

```diff
@@ -0,0 +1,65 @@
+# ------------------------------------------------------------------------------
+# Part of Red Queen Project.  This file is distributed under the MIT License.
+# See accompanying file /LICENSE for details.
+# ------------------------------------------------------------------------------
+
+"""Benchmark Bernstein Vazirani circuits."""
+
+import pytest
+
+from qiskit import QuantumCircuit, QuantumRegister, ClassicalRegister
+
+from applications import backends, run_qiskit_circuit
+
+
+def build_bv_circuit(secret_string, mid_circuit_measure=False):
```

Comment:
Is there a general plan for how we can support other compilers here? Do we want to prefer OpenQASM, or Qiskit circuits with converters, or should we be adding a copy for each compiler we want to support?

Reply:
The portability of qasm for benchmark circuit exchange between compilers seems more open. Also, it seems nice that no compiler occupies a preferred position in the framework if its something we want to encourage others to try.

Reply:
I've pivoted the benchmark to use QASM in: https://github.com/Qiskit/red-queen/pull/6/commits/e7e52198c178490cc48433957800822de352a064 I still use the same qiskit script to generate the circuit but the benchmarks are run from a saved qasm file